src/components/EditModal/EditModal.jsx

```diff
@@ -1,86 +1,96 @@
-import React, { useState } from 'react';
-import './EditModal.css'
-import { useSelector } from 'react-redux';
-import languageSelector from '../../assets/languages/languageSelector';
+import React, { useState } from "react";
+import "./EditModal.css";
+import { useSelector } from "react-redux";
+import languageSelector from "../../assets/languages/languageSelector";
 
 const EditModal = ({ item, onClose, handleUpdate }) => {
   const [title, setTitle] = useState(item.title);
   const [price, setPrice] = useState(item.price);
   const [description, setDescription] = useState(item.description);
   const [errors, setErrors] = useState({});
-  const language = useSelector(state=> state.languageReducer);
+  const language = useSelector((state) => state.languageReducer);
 
-
-
-
-  const handleSumbit = (e) =>{
+  const handleSumbit = (e) => {
     e.preventDefault();
     const validationErrors = {};
     const form = e.target;
     const title = form.title.value;
     const price = form.price.value;
     const description = form.description.value;
 
-    if (title.length <= 1 || typeof title !== 'string') {
-      validationErrors.title = 'title must be at least 2 characters long';
+    if (title.length <= 1 || typeof title !== "string") {
```

Comment:
`if (!title.length || typeof title !== "string") {`

Reply:
No, this is not done as expected, check the falsy value for `title.length`, check that the conditional should be:
`if (!title.length || typeof title !== "string") {`  => `!title.length`

Reply:
now it is resolved